fbchat/_client.py

```diff
@@ -976,6 +976,42 @@ def getUserActiveStatus(self, user_id):
         """
         return self._buddylist.get(str(user_id))
 
+    def _fetchImages(self, thread_id):
```

Comment:
Is there a reason for having the two methods? If not, I'd prefer only having one

Reply:
Its easier to manage code this way for me, but yeah, I'll merge them.